src/Domain/Model/MembershipApplication.php

```diff
@@ -30,24 +32,27 @@ class MembershipApplication {
 	private $needsModeration;
 	private $isCancelled;
 
-	public static function newApplication( string $type, MembershipApplicant $applicant, MembershipPayment $payment ): self {
+	public static function newApplication( string $type, MembershipApplicant $applicant, MembershipPayment $payment,
+										   MembershipApplicationTrackingInfo $trackingInfo ): self {
 		return new self(
 			null,
 			$type,
 			$applicant,
 			$payment,
+			$trackingInfo,
 			self::NO_MODERATION_NEEDED,
 			self::IS_CURRENT
 		);
 	}
 
 	public function __construct( int $id = null, string $type, MembershipApplicant $applicant, MembershipPayment $payment,
-		bool $needsModeration, bool $isCancelled ) {
+		MembershipApplicationTrackingInfo $trackingInfo, bool $needsModeration, bool $isCancelled ) {
 
 		$this->id = $id;
 		$this->type = $type;
 		$this->applicant = $applicant;
 		$this->payment = $payment;
+		$this->trackingInfo = $trackingInfo;
```

Comment:
You did not create the field. More importantly though, I'm surprised you are adding this here. I thought the plan was to have the tracking info handled by a service?